jdk/Dockerfile

```diff
@@ -0,0 +1,20 @@
+ARG TEMURIN_TAG
+
+FROM eclipse-temurin:$TEMURIN_TAG
+
+ARG TEMURIN_TAG
+MAINTAINER Dwolla Engineering <dev+docker@dwolla.com>
+LABEL org.label-schema.vcs-url="https://github.com/Dwolla/docker-java"
+
+RUN if [ "$TEMURIN_TAG" = "8u322-b06-jdk" ] ; then \
```

Comment:
This is clever! How difficult would it be to have it check whether `"$TEMURIN_TAG"` starts with `8u` instead of doing a full string match, so we don't have to worry about updating this whenever the tag is updated?

Reply:
good idea!